docs/src/pages/components/buttons/SplitButton.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import ClickAwayListener from '@material-ui/core/ClickAwayListener';
+import Grow from '@material-ui/core/Grow';
+import Paper from '@material-ui/core/Paper';
+import Popper from '@material-ui/core/Popper';
+import MenuItem from '@material-ui/core/MenuItem';
+import MenuList from '@material-ui/core/MenuList';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: '100%',
+    display: 'flex',
+  },
+  container: {
+    marginBottom: theme.spacing(2),
+  },
+  group: {
```

Comment:
Do we need this style?

Reply:
Dead (from before conversion of styles in the Button to the ButtonGroup wrapper).

Removed.